docs/github/continuous-integration.md

```diff
@@ -0,0 +1,284 @@
+# GitHub and Continuous Integration
+
+## Housekeeping
+
+-  Bookmark your project’s GitHub [repository](https://github.com/githubtraining/hellogitworld)
+-  Update your progress tracker
+-  Install [Git for Windows](https://git-scm.com/downloads) or [GitHub for Desktop](https://desktop.github.com/)
+-  Bookmark GitHub’s [Git Cheat Sheet](https://education.github.com/git-cheat-sheet-education.pdf)
```

Comment:
- Maybe rename this section to "Resources"
- move it to the end of the doc
- delete the first two bullets altogether

Reply:
In Taheer's other comment he said that first link is a placeholder that we would replace with the repo for each project... that is an extra step to remember when setting up a new project though (risk we might forget and leave it as this placeholder). Maybe we remove the link and just make sure they bookmark it during training?

About the progress tracker, I think this info comes from a doc used in training and that is about the training progress tracker. So that bullet can probably be removed.

Reply:
Updated section to be titled Resources and removed the first 2 bullet points.  Now that it is part greater git.md doc in learning section I'm not sure if it still makes sense to move this section to the end. Please take a look and let me know,